src/Twig/MarkdownExtension.php

```diff
@@ -174,4 +177,20 @@ public function formatValue($value)
     {
         return \is_array($value) ? json_encode($value) : $value;
     }
+
+    private function isExcluded(ClassParser $classParser, array $tagsToIgnore = [], array $excluded = []): bool {
+        foreach ($tagsToIgnore as $tagToIgnore) {
+            if ($classParser->hasTag($tagToIgnore)) {
+                return true;
+            }
+        }
+
+        foreach ($excluded as $excludePattern) {
+            if (function_exists('fnmatch') && fnmatch($excludePattern, $classParser->getFileName())) {
```

Comment:
Move `function_exists` around the foreach loop to prevent a useless foreach call if the function does not exist